draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1527,7 +1531,7 @@ filing system or remote ones accessed by http (http://domain/dir/... ).-->
         from Safe Retreat, since it does indicate an upper limit to the current capacity.</t>
 
         <t>At the point where all packets sent in the Unvalidated Phase have been either acknowledged
-        or have been declared lost, the sender updates ssthresh and enters the Normal Phase.
+        or have now been declared lost, the sender updates ssthresh and enters the Normal Phase.
```

Comment:
I would remove "have now been", the sentence then reads:
At the point where all packets sent in the Unvalidated Phase have been either acknowledged or declared lost, the [...]